index.ts

```diff
@@ -241,15 +241,18 @@ export const Channel = {
   messages: () => ({}),
   sendMessage: async (args, { self }) => {
     const { id } = self.$argsAt(root.guilds.one.channels.one);
+    const msg = {
+      content: args.content,
+      components: JSON.parse(args.components || "[]"),
+      embeds: JSON.parse(args.embeds || "[]"),
+    };
+    if (args.message_reference) msg["message_reference"] = args.message_reference;
+    if (args.allowed_mentions) msg["allowed_mentions"] = args.allowed_mentions;
```

Comment:
Add support for message reference (i.e. "reply to") and allowed mentions configuration.